linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ESFeatures.scala

```diff
@@ -164,6 +189,7 @@ final class ESFeatures private (
 
   override def toString(): String = {
     s"""ESFeatures(
+       |  esVersion = $esVersion,
```

Comment:
The fingerprint also needs to be updated (I really don't like that we need to do all this manually, it's just too easy to forget :().